ForumProject/communications/consumers.py

```diff
@@ -0,0 +1,135 @@
+import json
+import logging
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+from .utils import get_room, get_messages, get_online_users, get_user_first_name, \
+    add_user_to_online, remove_user_from_online, create_message
+
+
+class ChatConsumer(AsyncWebsocketConsumer):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(args, kwargs)
```

Comment:
The call to the superclass constructor appears incorrect. It should be `super().__init__(*args, **kwargs)`. The current implementation might cause errors because `arg`s and `kwargs` are being passed as tuple and dict objects rather than unpacked arguments.